common/src/stack/report-system/command/report/system/tests/test_backend_link.py

```diff
@@ -0,0 +1,45 @@
+import pytest
+import paramiko
+import socket
+from stack import api
+
+@pytest.mark.parametrize('backend',
+[host['host'] for host in  api.Call('list host', args=['a:backend'])]
+)
+class TestLinkUp:
+
+	# See if the interfaces stacki configures are actually up on the hosts
+
+		def test_link_status(self, backend):
+			errors = []
+			backend_ssh = paramiko.SSHClient()
+			backend_ssh.set_missing_host_key_policy(paramiko.AutoAddPolicy())
+
+			# Get all interfaces for this host
+			interfaces = [
+				host['interface'] for host in api.Call('list host interface ', args=['a:backend']) if host['host'] == backend
```

Comment:
I believe the backend name could just be passed as arguments to `list host interface` rather than having to iterate through all interfaces for all backends.